packages/components/src/Components/ConditionalFilter/ConditionalFilter.js

```diff
@@ -83,6 +83,8 @@ class ConditionalFilter extends Component {
                             {
                                 ActiveComponent && <SplitItem isFilled>
                                     <ActiveComponent
+                                        isFilterable={activeItem.isFilterable}
```

Comment:
Is there a reason for the `isFilterable` prop?

Reply:
no, I missed that it is in filterItems, thanks!